src/components/Header/Header.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useContext } from 'react';
+import { Link } from 'react-router-dom';
+import Container from '@components/Layout/Container';
+import { AuthContext } from '../../App';
+import { HeaderContainer, Logo, Login } from './Header.styles';
+
+const Header = () => {
+  const { isLoggedIn } = useContext(AuthContext);
+
+  return (
+    <HeaderContainer>
+      <Container
+        width="100%"
+        maxWidth="1024px"
+        justifyContent="space-between"
+        alignItems="center"
+        padding="10px 0"
+      >
+        <Logo href="/">
+          <img
+            src="https://gift-s.kakaocdn.net/dn/gift/images/m640/pc_gift_logo.png"
+            alt="카카오 선물하기"
+          />
+        </Logo>
+        {isLoggedIn ? (
+          <Link to="./my-account">내 계정</Link>
```

Comment:
```suggestion
          <Link to="/my-account">내 계정</Link>
```

사소하긴 한데 일관되게 path 스타일 relative, absolute 중에 일관되게 맞추는게 좋겠습니다.